### DISCUSSION TITLE
Alter table set schema not implemented

### DISCUSSION THREAD

**Discussion by Alex-Monahan** [duckdb/duckdb#10641] [Ideas]
A MotherDuck user requested the ability to change the schema of an existing table. This would be helpful to avoid dropping/recreating when moving a table to a new schema. This is a minimal repro to trigger the `not implemented` error.

```sql
create schema s1;
create schema s2;
create table s1.original as select 42;
alter table s1.original set schema s2;
``` 
```
Error: Not implemented Error: T_AlterObjectSchemaStmt
```

**Comment by rlancer** [comment]
We have a growing database with a lot of data sources and we want to organize them into schemas. We made a schema for some of the new tables but cant move the existing tables to a new schema

**Comment by Mickychen00** [comment]
Is there any updates for this feature?

**Comment by zachlatta** [comment]
Just want to chime in here this would be helpful for our use case too! DuckDB is amazing
